### PR TITLE
[T5A1][T14-B1] Tan Min Rong Samson

### PR DESCRIPTION
**Pull request by samsontmr**
Ready for review.

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -73,13 +74,32 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Change tag friends to buddies
+
+**MSS**
+
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: the activity was updated to rename a tag such that all people with the given tag will have the tag updated. please check the description again

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -73,13 +74,32 @@ Use case ends.
...
+  Use case resumes at step 1.
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
-4. Should favor DOS style commands over Unix-style commands
+4. Should favor DOS style commands over Unix-style commands.
+5. Should not be a resource hog.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: what resource? how do you quantify resource hog?
- in general, try to be as precise and objective as possible.
- words like nice, easy, more, less etc can be avoided and use precise values (where possible)

**Comment by okkhoy** [comment]
@samsontmr 
Some comments added. Please ack & close the PR after reading comments.

**Comment by samsontmr** [comment]
acknowledge, thanks!

**Pull request closed without merging**